src/Environment.js

```diff
@@ -46,6 +47,31 @@ class CookieStorage {
   };
 }
 
+export async function isThisLive() {
+  const storage = new Cookies();
+
+  let liveStatus = storage.get('liveStatus');
+
+  if (liveStatus !== undefined) {
+    return liveStatus;
+  } else {
+    const res = await axios.get(
+      'https://tmi.twitch.tv/group/user/bdougieyo/chatters',
+    );
+
+    const {chatters} = res.data;
+    liveStatus = chatters.viewers.length > 0;
+    const fewHours = new Date(Date.now() + 1000 * 60 * 60 * 4); // 4 hours
+
+    storage.set('liveStatus', liveStatus, {
+      path: '/',
+      secure: process.env.NODE_ENV === 'development' ? false : true,
+      sameSite: 'strict',
+      expires: fewHours,
+    });
```

Comment:
This is great but.... Since this repo already has a netlify function running and server caching... consider using that and a new function to check for the live stream.